src/Components/v2/CollectionsHubsNav.tsx

```diff
@@ -0,0 +1,58 @@
+import { CSSGrid } from "@artsy/palette"
```

Comment:
One day we need to figure out the `v2` folder story :/ Some new stuff goes in it, some in the parent. I'll add it to the list...

Reply:
I intentionally moved these components there after the Slack convo from a couple days ago. Should I not have?

Reply:
oh no its fine! I was just commenting on how there's no enforced consistency ATM. At least we can be 100% sure (at a glance) that components in v2 are new.